### ISSUE TITLE
if you type ---> ./a.out cd ../; /bin/ls | /usr/bin/grep *

### ISSUE THREAD

**Issue by jblackiex** [Glagan/42-exam-rank-04#3]
ls and grep don't execute themselves into the rigth dir, cd command doesn't work in this case indeed; is it ok?

**Comment by Glagan** [comment]
It's been a long time since I did this project but I tried quickly and the commands seems to be executed as you expect.  
Did you use the exact command you posted, or did you use one that the program should handle:

```
./a.out cd ../ ";" /bin/ls "|" /usr/bin/grep "*"
```

The only problem I see here is the glob pattern, which are not supported, maybe that's why you think the output is wrong ?  
If you execute the command above and compare it to ``cd ../ ; /bin/ls | /usr/bin/grep "*"`` you get the same output (notice the glob pattern is in quotes).  
If you don't quote the ``*`` you **do** get a different output, but it's because when you send it to microshell, the pattern is generated before the command execution, while when executing it directly, the pattern seems to be executed on the fly and so after the ``cd``, I think.
